lib/main_common.pm

```diff
@@ -1708,6 +1708,7 @@ sub load_extra_tests_console {
     loadtest 'console/osinfo_db' if (is_sle('12-SP3+') && !is_jeos);
     loadtest 'console/libgcrypt' if ((is_sle(">=12-SP4") && (check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk'))) || is_opensuse);
     loadtest "console/gd";
+    loadtest 'console/valgrind' if (!is_sle('<=12-SP3'));
```

Comment:
minor style nitpick. `if (!$condition)` is commonly written in perl as `unless $condition`. Also see the same some lines above

Reply:
... and another thing learned :slightly_smiling_face: 

    loadtest 'console/osinfo_db' if (is_sle('12-SP3+') && !is_jeos);
    loadtest 'console/libgcrypt' if ((is_sle(">=12-SP4") && (check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk'))) || is_opensuse);
    loadtest "console/gd";
    loadtest 'console/valgrind' unless is_sle('<=12-SP3');